github_helper.py

```diff
@@ -45,25 +56,15 @@ def get(self, endpoint):
             to https://api.github.com/repos/{self.org}/{self.repo}/
         Returns:
             JSON object retrieved from the endpoint.
-            None if there are none or an error is received.
+        Raises:
+            HTTPError if we get an HTTP error status.
```

Comment:
Should document all three kinds of errors. Or wrap them so that we always get a basic IOError with a more reasonable message, so that calling code just has to handle the IOError